packages/mds-db/geographies.ts

```diff
@@ -157,51 +181,75 @@ export async function publishGeography(params: PublishGeographiesParams) {
     const conditions = []
     conditions.push(`publish_date = ${vals.add(publish_date)}`)
     const sql = `UPDATE ${schema.TABLE.geographies} SET ${conditions} where geography_id=${vals.add(geography_id)}`
-
     await client.query(sql, vals.values())
-    return geography_id
+    const res = await readSingleGeography(geography_id)
+    return res
```

Comment:
I think you could do this with one query by doing an `UPDATE ... returning *` or something along those lines, instead of updating and then querying.